aws/api/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -0,0 +1,627 @@
+// +build !ignore_autogenerated
```

Comment:
The Kubebuilder docs list the fact that there is only one `zz_generate.deeepcopy.go` file created here instead of one for each AWS api kind. This seems like a very minor concern to me and it generally does not make grokking the codebase significantly easier IMO. Would like to hear others thoughts here.